app/src/main/java/edu/uc/zhukv/droneradarmap/Weather_Layer/TransparentTileOWM.kt

```diff
@@ -91,14 +101,4 @@ class TransparentTileOWM(private val tileType: String):TileProvider {
         private const val OWM_TILE_URL = "http://tile.openweathermap.org/map/%s/%d/%d/%d.png?appid=d6d46d84c231bd013c9f0088629b0eb8"
     }
 
-    /**
-     * This constructor assumes the `url` parameter contains three placeholders for the x- and y-positions of
-     * the tile as well as the zoom level of the tile. The placeholders are assumed to be `{x}`,
-     * `{y}`, and `{zoom}`. An example
-     * for an OpenWeatherMap URL would be: http://tile.openweathermap.org/map/precipitation/{zoom}/{x}/{y}.png
-     *
-     */
-    init {
```

Comment:
The constructor needed to be moved higher in the code
 per: https://kotlinlang.org/docs/coding-conventions.html#class-layout